terraform/README.md

```diff
@@ -16,9 +16,11 @@ And the following services provisioned:
 ## 📂 Setup
 Follow these steps to deploy the infrastructure:
 
-1. Clone the repository to your local machine.
+1. Clone the repository to your local machine if you haven't already.
 
-2. Create a `terraform.tfvars` file in this directory with your AWS credentials and infrastructure details:
+2. Perform the short and long term ETL setup by reading `../short_term/README.md` and `../long_term/README.md`
```

Comment:
What do you mean by reading here? Do you mean following the steps outlined in the [short term](../short_term/README.md) and [long term](../long_term/README.md) documentation?